demo/components/victory-tooltip-demo.js

```diff
@@ -172,4 +175,94 @@ class App extends React.Component {
   }
 }
 
+class VictoryTooltipInPortal extends VictoryTooltip {
+  static contextTypes = {
+    portalRegister: React.PropTypes.func,
+    portalDeregister: React.PropTypes.func
+  }
+
+  render() {
+    this.element = super.render();
+    return null;
+  }
+
+  componentDidUpdate() {
+    this.context.portalRegister(this, this.element);
+  }
+
+  componentWillUnmount() {
+    this.context.portalDeregister(this);
+  }
+}
+
+class SvgPortal extends React.Component {
+  componentWillMount() {
+    this.map = new Map();
```

Comment:
ah yes, that's also a possibility, but having children request keys for themselves has the advantage it can never go wrong, let's see, either solution should work